config/rbac/role.yaml

```diff
@@ -85,7 +85,6 @@ rules:
   - deployments
   - replicasets
   - statefulsets
-  - controllerrevisions
```

Comment:
Do we need this for upgradeController?

Reply:
i think its missing somewhere in annotation, i didnt actively remove it, in any case lemme revet this one

Reply:
Fixed in #505